### PR TITLE
fix(sqs inbound): add try catch for correlate method

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

add Try catch to correlate method

### CODE REVIEW

File: connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java
Comment (sbuettner) [reply connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java:51]: @Oleksiivanov We dont delete messages that failed correlation because of an expception but we delete them in case of failed acitvation. If you read the comments from @chillleader you can see that we dont requeue messages that were not activated:
> NACK (no requeue) - inbound event not correlated
getChannel().basicReject(envelope.getDeliveryTag(), false);

In the case of a failed correlation because of an exeption we do requeue: 
> NACK (requeue) - failed to correlate event", e);
getChannel().basicReject(envelope.getDeliveryTag(), true);

File: connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java
Comment (igpetrov) [inline connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java:55]: One exception statement says `no requeue`, the other one `requeue` but they both do nothing. Is that expected?

File: connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java
Comment (igpetrov) [inline connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java:68]: It was correlated but not activated.

**Review by igpetrov — APPROVED** [review]

**Comment by igpetrov** [comment]
Let's wait for @sbuettner for one more review round

File: connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java
Comment (sbuettner) [inline connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java:48]: Please include the `receiveMessage` in the try catch block. Otherwise the connector stops working when there is a network error for example.

**Review by sbuettner — CHANGES_REQUESTED** [review]
Please change the scope of the `try ... catch`.

**Review by sbuettner — APPROVED** [review]

**Pull request merged**
